### ISSUE TITLE
Meaning of "self" in sequence diagrams

### ISSUE THREAD

**Issue by tswuxia** [nus-cs2103-AY1920S1/forum#189]
![image](https://user-images.githubusercontent.com/51690889/69539033-a2ef0e00-0fbe-11ea-9597-2ab9259b813e.png)
![image](https://user-images.githubusercontent.com/51690889/69539051-aedad000-0fbe-11ea-9e09-a8ed577aede2.png)

Why does the comment say m2 calls **m4 of self**? I thought self should refer to f:Foo instead of b:Bar? Or is it I misunderstood the meaning of "self" wrongly. Thanks.

**Comment by damithc** [comment]
> Or is it I understood the meaning of "self" wrongly.

Yes. Here, it means an object calling a method of itself. There is only one such case in the the diagram.
